From 8cf057b2d09e1831c422521b9d63f1e84ece036a Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@ubuntu.com>
Date: Mon, 13 Jan 2014 12:13:34 +0000
Subject: Don't run gettext_strings_test

This test is mainly useful as an upstream maintenance check.

Forwarded: not-needed
Last-Update: 2013-12-23

Patch-Name: skip_gettext_strings_test.patch
---
 tests/gettext_strings_test.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/gettext_strings_test.in b/tests/gettext_strings_test.in
index 5c305e7..15405cf 100644
--- a/tests/gettext_strings_test.in
+++ b/tests/gettext_strings_test.in
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+echo "Skipping upstream maintenance check."
+exit 77
+
 cd '@srcdir@'
 
 tdir="$(mktemp -d "${TMPDIR:-/tmp}/tmp.XXXXXXXXXX")"
